CONTRIBUTING.md

```diff
@@ -55,7 +55,15 @@ We will review your PR as soon as possible, and your contribution will be greatl
 Most likely, you'll need to create new secondary entry point to put the new utility in. To create entry point, use the following command:
 
 ```shell
-pnx nx g local-plugin:entry-point <name-of-your-utility> --library=ngxtension --skip-module
+pnpm nx g local-plugin:entry-point <name-of-your-utility> --library=ngxtension --skip-module
```

Comment:
```suggestion
pnpm exec nx g local-plugin:entry-point <name-of-your-utility> --library=ngxtension --skip-module
```

Reply:
It works without the `exec` ?

Reply:
Yeah but I'd rather be consistent with the commands.